deployment/roles/monitor/tasks/jumpbox.yml

```diff
@@ -0,0 +1,12 @@
+- name: Launch container
```

Comment:
what exactly is this tasks doing? When is it going to be run? Once during deployment?  Periodically by cron? Could you clarify?

Reply:
1. The `Launch container` launches the monitor server, serving data over HTTP GET requests. It is started during deployment, and setup to start after reboot and available for administration as a `tokenbridge-monitor` service - much like `poabridge` and `tokenbridge-ui` services.
2. The data, served by the above server, is produced by running `checkWorker.js` and `checkWorker2.js`
    - I run it initially in `Initial checks ...` steps because otherwise, the monitor server breaks when there's no data to serve
    - These scripts should be run periodically by cron - I created #136 and hope to do it in separate PR

Reply:
thanks. Is my understanding correct and the monitor server cannot start without the data produces by workers? Does it mean that the proper way to fix it is to run workers before starting the server. Imagine the situation when we restart `tokenbridge-monitor`, it will reset the file system and no data will exist for the monitor server, so the restart will be unsuccessful.

So, could you consider to remove `Initial checks 1/2` and `Initial checks 2/2` but add running of the workers in the start script of `package.json`?
https://github.com/poanetwork/tokenbridge/blob/f4151230ca9c678932d6ca0a0737ada97b69346d/monitor/package.json#L8

Reply:
You're spot on.
I'll do the changes.